pkgs/servers/jellyfin/web.nix

```diff
@@ -1,41 +1,44 @@
-{
-  lib,
-  stdenv,
-  overrideSDK,
-  fetchFromGitHub,
-  buildNpmPackage,
-  nix-update-script,
-  pkg-config,
-  xcbuild,
-  pango,
-  giflib,
-  darwin,
```

Comment:
Did you use `nixfmt`? There is no consensus in Nixpkgs about it's formatting rules, and this change enlarges the diff for no good reason.

Reply:
> Did you use `nixfmt`? There is no consensus in Nixpkgs about it's formatting rules, and this change enlarges the diff for no good reason.

The outdated code that you're commenting on was formatted by whatever `nixd` uses. Now, I  used `nixfmt-rfc-style` on both files and rebased. The usage of `nixfmt-rfc-style` appears to be encouraged, so I'll keep it like this.